packages/utils/src/crypto.ts

```diff
@@ -129,3 +129,18 @@ export const genEpochTreeLeaf = (
     }
     return polysum
 }
+
+export const genNullifier = (
+    identityNullifier: bigint,
+    externalNullifier: number | bigint
+): bigint => {
+    return hash2([BigInt(identityNullifier), BigInt(externalNullifier)])
+}
+
+export const genProposalId = (
```

Comment:
I think this is named incorrectly, this looks like an identity commitment calculation?